### PR TITLE
Added encoder mode11 to tune 1(OQ)

### PR DESCRIPTION
**Pull request by OmranAbdelkafi**
- Added M11 to OQ mode (tune1) by applying some M12 SQ (tune 0) features . Adopted features as follow : 

* M12 SQ ME search region .
* M12 SQ chroma level.
* M12 SQ Subpel Parameters .
* M12 SQ Partitioning 
* M12 SQ pf MD parameters.
* M12 SQ ADP level.
* Tune a new level for CU16x16 for better speed / BD-rate tradeoff.

=> M11 tune 1 : Speed = ~189 fps for 4K_60 on 8180 / BD-rate loss compared to M10 Tune1 : ~ 9 % 

- Added the ability to enable/disable BitRateReduction and Sharpness features for tune 1

### CODE REVIEW

**Comment by OmranAbdelkafi** [comment]
> some formatting comments:
> 
> EbEncDecProcess.c -
> 2792 remove blank line
> 2898 - extra semicolon at end
> 
> EbModeDecision.c
> 1234 - are { lined up correctly?
> 
> EbPictureDecisionProcess.c
> 378 - remove blank line
> 617 - remove blank line

**Comment by OmranAbdelkafi** [comment]
> some formatting comments:
> 
> EbEncDecProcess.c -
> 2792 remove blank line
> 2898 - extra semicolon at end
> 
> EbModeDecision.c
> 1234 - are { lined up correctly?
> 
> EbPictureDecisionProcess.c
> 378 - remove blank line
> 617 - remove blank line

fixed within the latest commit

**Comment by OmranAbdelkafi** [comment]
> Also can you recommend/make changes to the user guide regarding these parameters?
> -tune
> -brr
> -sharp
> 
> I believe we were going to allow tune 0 to be used, but document that it is being depreciated.

Updated the user guide concerning brr and sharp as they could be used for all tune now

**Comment by tianjunwork** [comment]
@OmranAbdelkafi Could you also update this table? Thx. Docs/encoding_mode_res_table.png

File: Source/Lib/Codec/EbEncHandle.c [hunk 1]
```diff
@@ -2906,7 +2891,7 @@ static void PrintLibParams(
     else
         SVT_LOG("\nSVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange\t\t\t: CQP / %d / %d / %d ", config->qp, config->lookAheadDistance, config->sceneChangeDetection);
 
-    if (config->tune == 0)
+    if (config->tune <= 1)
```
Comment (tianjunwork) [inline Source/Lib/Codec/EbEncHandle.c]: tune 2 also applies?

File: Source/Lib/Codec/EbDefinitions.h [hunk 2]
```diff
@@ -2049,7 +2049,7 @@ static const EB_U8 SearchAreaHeightOq[5][MAX_SUPPORTED_MODES] = {
 	{  64,   64,   64,   64,   16,   13,   13,    9,    9,    7,    7,    7,    7 },
 	{  64,   64,   64,   64,   16,    9,    9,    7,    7,    7,    7,    7,    7 },
 	{  64,   64,   64,   64,   16,   13,   13,    9,    9,    7,    7,    7,    7 },
-	{  64,   64,   64,   64,    9,    9,    9,    9,    7,    7,    7,    7,    7 }
+	{  64,   64,   64,   64,    9,    9,    9,    9,    7,    7,    7,    5,    5 }
 };
```
Comment (tianjunwork) [inline Source/Lib/Codec/EbDefinitions.h]: {  64,   64,   64,   64,   24,   16,   13,    9,    9,    7,    7,    7,    5 } is the SQ config.
Is this a typo?

**Pull request merged**